src/plugins/dataflow/nodes/dataflow-node-plot.tsx

```diff
@@ -77,7 +124,11 @@ function lineData(node: any) {
   return chartData;
 }
 
-function lineOptions() {
+
+function lineOptions(node: any) {
+  const max = node.data.tickMax || node.data.dsMax;
+  const min = node.data.tickMin || node.data.dsMin;
```

Comment:
```suggestion
  const max = node.data.tickMax ?? node.data.dsMax;
  const min = node.data.tickMin ?? node.data.dsMin;
```
Since this code is repeated, you might consider making a separate function for it, then calling it here and on lines 34 and 35.

Reply:
Done.